packages/@aws-cdk/core/test/arn.test.ts

```diff
@@ -165,6 +170,17 @@ nodeunitShim({
           resource: 'identitypool',
           resourceName: 'us-east-1:1a1a1a1a-ffff-1111-9999-12345678:bla',
           sep: '/',
+          checkCfnEncoding: false,
```

Comment:
Wait what? Why `false`? Is this a bug that needs fixing?

Reply:
Yes, but I don't know how to fix it 😕.

Basically, the `:` inside the `resourceName` make it impossible to handle using CFN functions (we would need things like `Fn::Length`, `Fn::SubArray`, etc. to be able to handle this).

Reply:
Ah yeah.

Maybe put that in there as a comment.

Reply:
Done.